### DISCUSSION TITLE
wasm_bindgen::module is currently only supported with `--target no-modules` and `--target web`

### DISCUSSION THREAD

**Discussion by Codetrauma** [wasm-bindgen/wasm-bindgen#3904] [Q&A]
Followed documentation and have a project currently working in the browser, but when trying to build into nodejs I get this error. I couldn't find anything in previous issues about this, wondering if their is a solution already or any documentation on this. Thank you.

https://github.com/tlsnotary/tlsn-js/tree/main/wasm/prover

Works with: ""wasm-pack build --target web wasm/prover"

but get the above error with: "wasm-pack build --target nodejs wasm/prover"

**Comment by daxpedda** [comment]
It would probably be helpful to at least post the error message.

A minimal example, not using `wasm-pack`, would also be appropriate considering you are filing this at `wasm-bindgen` and not `wasm-pack`.
